### PR TITLE
powdenest before simplify in gruntz - issue #11879

### LINKED ISSUE #11879: Strange output from common limit used in elementary calculus
I notice that the following limit expression returns output that appears to be wrong:

```python
x, h = symbols("x h")
n = symbols('n', integer=True)
limit(((x+h)**n-x**n)/h, h, 0)
```

image

Here's [Wolfram's result](https://goo.gl/0EiVO0)

### PR DESCRIPTION
**Pull request by bsdz**
As suggested by @jksuom, add a powdenest before simplify. Also check expression has exp and log before denesting. This might not be necessary.
 
Fixes #11879

### CODE REVIEW

File: sympy/series/tests/test_limits.py [hunk 1]
```diff
@@ -477,7 +477,9 @@ def test_limit_seq():
     assert (limit(Sum(y**2 * Sum(2**z/z, (z, 1, y)), (y, 1, x)) /
                   (2**x*x), x, oo) == 4)
 
-
+def test_issue_11879():
+    assert simplify(limit(((x+y)**n-x**n)/y, y, 0)) == n*x**(n-1)
+
```
Comment (jksuom) [inline sympy/series/tests/test_limits.py]: Travis finds trailing whitespace here.

**Comment by jksuom** [comment]
This looks good. Checking for exp and log will probably save time in most cases. Thanks!

**Comment by smichr** [comment]
ping @bsdz , I am updating the AUTHORS file and am wondering if it OK to list your name as "Blair Azzopardi" instead of "bsdz". I'm not sure how to contact you so am doing so here.

**Comment by bsdz** [comment]
Yes. That's okay. Thank you.

**Comment by smichr** [comment]
> Yes. That's okay. Thank you.

Thanks for the response. I'll get that updated.

**Pull request merged**